samples/apps/txregulator/clients/loader.py

```diff
@@ -23,7 +23,7 @@ def __init__(self, network, name, country):
         network.consortium.add_users(primary, [self.name])
 
         with primary.user_client(user_id=self.name) as client:
-            self.ccf_id = client.rpc("whoAmI", {}).result["caller_id"]
+            self.ccf_id = client.get("whoAmI").result["caller_id"]
```

Comment:
We can probably scrap the result/error fields too.

Reply:
We could, but it may be useful to keep the distinction as they generally have different types (`result` is a dict-built-from-JSON, `error` is a string). I think ideally this response would look like (or even be?) a `requests.Response` object, so the caller could access the status/headers/contents in a standard way.

Reply:
yes, that's what I meant. For the curl client, we might have to construct the Response slightly, but that shouldn't be too bad. I'll take a look at that.